server/package.json

```diff
@@ -43,6 +41,8 @@
     "source-map-support": "^0.5.10"
   },
   "devDependencies": {
+    "@google-cloud/compute": "^0.10.0",
```

Comment:
These are actually used at runtime by the node server.

Reply:
oops.  correct.

Reply:
Reverted.